.github/workflows/zero-downtime-tests.yml

```diff
@@ -1,13 +1,450 @@
+          ls -la ./keptn*.tgz
+          echo "UPGRADE_HELM_CHART=$(ls './keptn-${{ github.event.inputs.upgradeTo }}.tgz')" >> $GITHUB_ENV      
+          echo "INSTALL_HELM_CHART=$(ls ./keptn-installed.tgz)" >> $GITHUB_ENV
+
+      - name: Install and start GKE cluster
+        if: env.CLOUD_PROVIDER == 'GKE'
+        env:
+          GKE_VERSION: ${{ matrix.PLATFORM_VERSION }}
+          GCLOUD_SERVICE_KEY: ${{ secrets.GCLOUD_RESTRICTED_SERVICE_KEY }}
+          GCLOUD_PROJECT_NAME: ${{ secrets.GCLOUD_PROJECT_NAME }}
+          CLOUDSDK_COMPUTE_ZONE: "us-east1-b"
+          CLOUDSDK_REGION: "us-east1"
+          GKE_KUBECONFIG: ${{ secrets.INTEGRATION_TEST_KUBECONFIG }}
+          BRANCH: ${{ steps.determine_branch.outputs.BRANCH}}
+        id: install_gke
+        run: |
+          export CLUSTER_NAME_NIGHTLY=${{ matrix.CLUSTER_NAME }}
+          echo "Setting up GCloud CLI"
+          export OS_TYPE="linux"
+          
+          echo ${GCLOUD_SERVICE_KEY} | base64 --decode > ~/gcloud-service-key.json
+          export GOOGLE_APPLICATION_CREDENTIALS=~/gcloud-service-key.json
+          export CLOUDSDK_CORE_DISABLE_PROMPTS=1;
+          
+          gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+          gcloud --quiet config set project "$GCLOUD_PROJECT_NAME"
+          gcloud --quiet config set container/cluster "$CLUSTER_NAME_NIGHTLY"
+          gcloud --quiet config set compute/zone "${CLOUDSDK_COMPUTE_ZONE}"
+          
+          echo "GOOGLE_APPLICATION_CREDENTIALS=${GOOGLE_APPLICATION_CREDENTIALS}" >> $GITHUB_ENV
+          
+          echo "Setting up kubectl"
+          echo "Setting KUBECONFIG to $PWD/.kubeconfig"
+          
+          echo "$GKE_KUBECONFIG" >> "$PWD/.kubeconfig"
+          chmod 600 "$PWD/.kubeconfig"
+          export KUBECONFIG="$PWD/.kubeconfig"
+          
+          echo "KUBECONFIG=${KUBECONFIG}" >> $GITHUB_ENV
+          
+          echo "Setting kube context..."
+          kubectl config use-context "$CLUSTER_NAME_NIGHTLY"
+          
+          echo "##[set-output name=CLUSTER_NAME_NIGHTLY;]$(echo ${CLUSTER_NAME_NIGHTLY})"
+
+      - name: Install Keptn
+        id: keptn_install
+        timeout-minutes: 10
+        env:
+          HELM_CHART_NAME: ${{ steps.extract_helm_chart_name.outputs.HELM_CHART_NAME }}
+          USE_RESOURCE_SERVICE: ${{ matrix.USE_RESOURCE_SERVICE }}
+        run: |
+          echo "Installing Keptn on cluster"
+          echo "{}" > creds.json # empty credentials file
+          
+          echo "::group::Keptn Installation Log"
+          
+          # Use Keptn helm chart to be able to customize the values
+          helm install -n ${KEPTN_NAMESPACE} keptn ${HELM_CHART_NAME} \
+            --create-namespace \
+            --set control-plane.apiGatewayNginx.type=${KEPTN_SERVICE_TYPE} \
+            --set control-plane.resourceService.enabled=${USE_RESOURCE_SERVICE} \
+            --set continuous-delivery.enabled=true \
```

Comment:
since this variable doesn't have a dynamic value, you could set it in the values file instead of here